chainerx/_docs/routines.py

```diff
@@ -498,6 +498,41 @@ def _docs_linalg():
 .. seealso:: :func:`numpy.dot`
 """)
 
+    _docs.set_doc(
+        chainerx.linalg.solve,
+        """solve(a, b)
+Solves a linear matrix equation, or system of linear scalar equations.
+
+It computes the exact solution of ``x`` in ``ax = b``,
+where ``a`` is a square and full rank matrix.
+
```

Comment:
Let's also document the possible shapes of "b".

Reply:
Done.